mqtt/mqtt-bridge/Cargo.toml

```diff
@@ -35,6 +35,7 @@ mqtt3 = { path = "../mqtt3", features = ["serde1"] }
 mqtt-broker = { path = "../mqtt-broker" }
 mqtt-util = { path = "../mqtt-util" }
 rustc-hash = "1.1.0"
+crc32fast = "1.2.1"
```

Comment:
are we settling on this less popular but faster (as author claims) crate? I don't see a problem per se, but we just need to make sure it works well on all platforms we support, cause it looks like it uses specific SIMD instructions.

Reply:
? it has 12 million downloads: https://crates.io/search?q=checksum&sort=downloads. I picked it because it was so popular :)

Reply:
Will order deps :)

Reply:
Also, I think this means it will not use simd for our use cases (we don't use nightly):

```rust
#![cfg_attr(not(feature = "std"), no_std)]
#![cfg_attr(
    all(feature = "nightly", target_arch = "aarch64"),
    feature(stdsimd, aarch64_target_feature)
)]
```

If everything works fine, we can just stick to this version as long as there are not major issues with it. Otherwise we can swap to something else :)